core/src/main/kotlin/br/com/concrete/yosef/api/property/size/DimenSpec.kt

```diff
@@ -0,0 +1,9 @@
+package br.com.concrete.yosef.api.property.size
+
+import android.view.View
+import br.com.concrete.yosef.entity.DynamicProperty
+
+interface DimenSpec {
+
+    fun applyDimenSpec(view: View, dynamicProperty: DynamicProperty)
+}
```

Comment:
Can you add one line here?

Reply:
Ok, I saw the same request on detekt report hasuahe